### PR TITLE
Added Scroll-to-top button

### PR DESCRIPTION
**Pull request by digambar-t7**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/921"></a>

### CODE REVIEW

**Comment by schmelto** [comment]
I think on mobile the button is a little bit to big. 

image

File: package.json [hunk 1]
```diff
@@ -90,7 +90,9 @@
   },
   "lint-staged": {
     "*.{js,css,md,json}": [
-      "prettier --write",
+      "prettier --write"
+    ],
+    "*.{js,md,json}": [
```
Comment (krishguptadev) [inline package.json:95]: Not sure why was this changed?

```suggestion
      "prettier --write",
```

Comment (krishguptadev) [reply package.json:95]: esling goes through css files? Also the ss isn't working

Comment (krishguptadev) [inline package.json:95]: If that is the case, I have another suggestion remove `md`

```suggestion
    "*.{js,json}": [
```

Comment (digambar-t7) [reply package.json:95]: Ok I will remove it .
By the way I had discussed about this error with one more person in detail. You can have a look at it here https://github.com/EddieHubCommunity/LinkFree/issues/912

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
It's a great start but a few things need fixing. My suggestions:
- Make the button smaller on all screen sizes
- Make the arrow white, at the moment the profile behind sometimes shows through
- Don't overlap the profile chips, the only time this might be slightly necessary is on mobile otherwise there is enough space to the right to avoid overlap

**Review by emmalearnscode — APPROVED** [review]
Looks much better!

**Review by schmelto — APPROVED** [review]
https://user-images.githubusercontent.com/30869493/148251552-0d1a632d-7e7c-4945-b8eb-cf04ebe0911c.MOV

File: package.json (hunk 1 above)
Comment (digambar-t7) [reply package.json]: Hey @krishguptadev, I have made the suggested change.

**Comment by ssensalo** [comment]
The button's color looks really strong on the page maybe we could make it a little lighter such as reducing the black/ darkness.
What do you think @emmalearnscode @schmelto @krishguptadev

**Comment by schmelto** [comment]
> The button's color looks really strong on the page maybe we could make it a little lighter such as reducing the black/ darkness.
> 
> What do you think @emmalearnscode @schmelto @krishguptadev 

Sounds like a good idea

File: src/Components/ScrollToTopBtn.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth',
+    })
+    setScrollBtnVisibility(scrollBtnVisibility)
+  }
+
+  return (
+    <BsFillArrowUpCircleFill
+      style={scrollBtnVisibility ? {} : { opacity: '0' }}
```
Comment (nathgoutam93) [inline src/Components/ScrollToTopBtn.js:33]: Not only opacity but also pointed event should be none.
So that it's not clickable while not visible

Comment (digambar-t7) [reply src/Components/ScrollToTopBtn.js]: Correct

**Comment by digambar-t7** [comment]
I have also changed the color of the btn such that it doesn't conflicts with the chips.
Do have a look at it.

**Review by krishguptadev — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]
